activities2Graph.py

```diff
@@ -120,7 +125,10 @@ def buildEdgesfromActivities(nodes, activities):
             new_nodes[activity['synId']]  = activity
             #Add input relationships
             for used in activity['used']:
-                edges = addNodesandEdges(used, nodes, activity, edges)
+                try:
```

Comment:
move try/catch inside of `addNodesandEdges` - around specific place(s) where there are errors. consider logging errors to a static file as well as the console (stderr)